selfdrive/logcatd/logcatd_android.cc

```diff
@@ -1,68 +1,62 @@
-#include <unistd.h>
-#include <cstdio>
-#include <cstdlib>
-#include <cassert>
-#include <csignal>
-#include <cerrno>
-
 #include <android/log.h>
 #include <log/logger.h>
 #include <log/logprint.h>
 
-#include "common/timing.h"
 #include "common/util.h"
 #include "messaging.hpp"
 
+ExitHandler do_exit;
+log_time last_log_time = {};
 
-int main() {
-  ExitHandler do_exit;
-
+static void publish_log(PubMaster &pm) {
```

Comment:
let's just keep this in the main function, should be simpler and we don't need the global